src/cascadia/TerminalApp/App.cpp

```diff
@@ -1451,6 +1459,153 @@ namespace winrt::TerminalApp::implementation
+                                   const TerminalApp::ActionEventArgs& args)
+    {
+        _SplitVertical(std::nullopt);
+        args.Handled(true);
+    }
+
+    void App::_HandleSplitHorizontal(const IInspectable& /*sender*/,
+                                     const TerminalApp::ActionEventArgs& args)
+    {
+        _SplitHorizontal(std::nullopt);
+        args.Handled(true);
+    }
+
+    void App::_HandleScrollUpPage(const IInspectable& /*sender*/,
+                                  const TerminalApp::ActionEventArgs& args)
+    {
+        _ScrollPage(-1);
+        args.Handled(true);
+    }
+
+    void App::_HandleScrollDownPage(const IInspectable& /*sender*/,
+                                    const TerminalApp::ActionEventArgs& args)
+    {
+        _ScrollPage(1);
+        args.Handled(true);
+    }
+
+    void App::_HandleOpenSettings(const IInspectable& /*sender*/,
+                                  const TerminalApp::ActionEventArgs& args)
+    {
+        _OpenSettings();
+        args.Handled(true);
+    }
+
+    void App::_HandlePasteText(const IInspectable& /*sender*/,
+                               const TerminalApp::ActionEventArgs& args)
+    {
+        _PasteText();
+        args.Handled(true);
+    }
+
+    void App::_HandleNewTabWithProfile(const IInspectable& /*sender*/,
+                                       const TerminalApp::ActionEventArgs& args)
+    {
+        if (const auto& realArgs = args.ActionArgs().try_as<TerminalApp::NewTabWithProfileArgs>())
+        {
+            _OpenNewTab({ realArgs.ProfileIndex() });
+            args.Handled(true);
+        }
+    }
+
+    void App::_HandleSwitchToTab(const IInspectable& /*sender*/,
+                                 const TerminalApp::ActionEventArgs& args)
+    {
+        if (const auto& realArgs = args.ActionArgs().try_as<TerminalApp::SwitchToTabArgs>())
+        {
+            _SelectTab({ realArgs.TabIndex() });
+            args.Handled(true);
+        }
+    }
+
+    void App::_HandleResizePane(const IInspectable& /*sender*/,
+                                const TerminalApp::ActionEventArgs& args)
+    {
+        if (const auto& realArgs = args.ActionArgs().try_as<TerminalApp::ResizePaneArgs>())
+        {
+            _ResizePane(realArgs.Direction());
+            args.Handled(true);
+        }
+    }
+
+    void App::_HandleMoveFocus(const IInspectable& /*sender*/,
+                               const TerminalApp::ActionEventArgs& args)
+    {
+        if (const auto& realArgs = args.ActionArgs().try_as<TerminalApp::MoveFocusArgs>())
+        {
+            _MoveFocus(realArgs.Direction());
+            args.Handled(true);
+        }
+    }
+
+    void App::_HandleCopyText(const IInspectable& /*sender*/,
+                              const TerminalApp::ActionEventArgs& args)
+    {
+        if (const auto& realArgs = args.ActionArgs().try_as<TerminalApp::CopyTextArgs>())
+        {
+            args.Handled(_CopyText(realArgs.TrimWhitespace()));

```

Comment:
eh this is the only one that does a nested call like this; nit: break out for clarity?